src/test/scala/edu/ie3/simona/test/common/input/EvcsInputTestData.scala

```diff
@@ -34,13 +35,54 @@ trait EvcsInputTestData extends DefaultTestData with NodeInputTestData {
     true,
```

Comment:
Please let's fix the power factor of this one to 1.0 (or 0.997 or something very close to one, if the purpose is to test reactive power as well)